src/bin/exrinfo/CMakeLists.txt

```diff
@@ -2,10 +2,17 @@
 # Copyright Contributors to the OpenEXR Project.
 
 add_executable(exrinfo main.c)
-target_link_libraries(exrinfo OpenEXR::OpenEXRCore)
+target_link_libraries(exrinfo OpenEXR::OpenEXRCore Imath::Imath)
```

Comment:
I really don't think this is right.  A library/application that says "I depend on OpenEXRCore" shouldn't be required to also say, "So I also depend on this other stuff that OpenEXRCore depends on". This applies to the extra `target_include_directories` statement below.

The wrinkle introduced by this PR is that a public header from OpenEXRCore (`openexr_config.h`) now depends on Imath (`ImathConfig.h`), so OpenEXRCore's dependency on Imath is no longer private, it needs to be passed to downstream applications. 

Shouldn't the problem should be resolved by adding `DEPENDENCIES Imath::Imath` to OpenEXRCore itself?

Reply:
yes, make sense.